services/gitlab/gitlab-merge-requests.spec.js

```diff
@@ -0,0 +1,36 @@
+import { test, given } from 'sazerac'
+import GitlabMergeRequests from './gitlab-merge-requests.service.js'
+
+describe('GitlabMergeRequests', function () {
+  test(GitlabMergeRequests.render, () => {
+    given({ variant: 'open', mergeRequestCount: 1399 }).expect({
+      label: 'merge requests',
+      message: '1.4k open',
+      color: 'brightgreen',
+    })
+    given({ variant: 'open', raw: '-raw', mergeRequestCount: 1399 }).expect({
+      label: 'open merge requests',
+      message: '1.4k',
+      color: 'brightgreen',
+    })
+    given({
+      variant: 'open',
+      labels: 'discussion,enhancement',
+      mergeRequestCount: 15,
+    }).expect({
+      label: 'discussion,enhancement merge requests',
+      message: '15 open',
+      color: 'brightgreen',
+    })
+    given({ variant: 'open', mergeRequestCount: 0 }).expect({
+      label: 'merge requests',
+      message: '0 open',
+      color: 'blue',
+    })
+    given({ variant: 'open', mergeRequestCount: 10001 }).expect({
+      label: 'merge requests',
+      message: 'more than 10k open',
+      color: 'brightgreen',
+    })
+  })
+})
```

Comment:
Let's also add a test that validates the authentication integration (especially since we recently found we'd missed a couple classes). I think you should be able to copy in the the test from one of the other GitLab services like Tag

https://github.com/badges/shields/blob/master/services/gitlab/gitlab-tag.spec.js

and just change the import class to be the MergeRequest service class instead